src/install/aws-logs/aws_services/configure-log-forwarding.mdx

```diff
@@ -3,7 +3,7 @@ headingText: Configure logs forwarding
 componentType: default
 ---
 
-There are multiple ways to send New Relic logs from your AWS services. Choose the path that best suits your needs. Both the integration instructions and the CloudFormation stack will adapt based on your selections.
+There are multiple ways to send New Relic logs from your AWS services. Choose the path that best suits your needs. Both the integration instructions and the CloudFormation stack will adapt to your selections.
```

Comment:
Suggestion: There are multiple ways to send  logs from your AWS services to New Relic.